jbmc/src/java_bytecode/java_bytecode_convert_class.cpp

```diff
@@ -118,11 +118,43 @@ class java_bytecode_convert_classt:public messaget
   // see definition below for more info
   static void add_array_types(symbol_tablet &symbol_table);
 
+  /// Check if a method is an overlay method by searching for
+  /// `ID_overlay_method` in its list of annotations.
+  ///
+  /// An overlay method is a method with the annotation
+  /// \@OverlayMethodImplementation. They should only appear in
+  /// [overlay classes](\ref java_class_loader.cpp::is_overlay_class). They
+  /// will be loaded by JBMC instead of the corresponding method in the
```

Comment:
:bee: Could you explain what "the corresponding method" is (i.e. method with the same signature in a class of the same name that was loaded before)?